build/illume.js

```diff
@@ -11,17 +11,15 @@ var _q = require("@standard-library/q");
 var _kefir = require("kefir");
 
 var offsetAbove = function offsetAbove(y, element) {
-  return y > $(element).offset().top;
+  return y > element.offsetTop;
```

Comment:
Though I think that I'm taking you on a wild goose chase with that link. Best to limit changes in this PR, but I'd accept the addition of a big integration test as appropriate to include. The configuration can be copied from the `dom-clicker` library above (`faucet`, `tape`, and `tap-run` dependencies).